napalm_logs/config/junos/OSPF_NEIGHBOR_DOWN.yml

```diff
@@ -0,0 +1,36 @@
+# This error tag corresponds to syslog messages notifying that the configured
+# ospf neighbor has changed state from Full
+messages:
+  # 'error' should be unique and vendor agnostic. Currently we are using the JUNOS syslog message name as the canonical name.
+  # This may change if we are able to find a more well defined naming system.
+  - error: OSPF_NEIGHBOR_DOWN
+    tag: RPD_OSPF_NBRDOWN
+    values:
+      neighbor: ([\w\d:\.]+)
+      interface: ([\w\-\/\.\d]+)
+      area: (\d+\.\d+\.\d+\.\d+)
+      reason_message: ([\w ]+)
+    line: 'OSPF neighbor {neighbor} (realm ospf-v2 {interface} area {area}) state changed from Full to Down due to KillNbr (event reason: {reason_message})'
+    model: openconfig-network-instance
```

Comment:
I missed this previously: I think we should reference the `openconfig-opsf`, as it's a standalone model (although it's just based on the network instance one). Thoughts?

Reply:
Makes sense, except for why openconfig does this :)